stl/STL.natvis

```diff
@@ -0,0 +1,1596 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
This doesn't affect the stl code or building, so I recommend moving it to a parallel folder. I recommend `<root>/debugger`.

Reply:
(FWIW, I'm happy with the now-current `stl/STL.natvis`.)